tools/minfi/minfi_read450k.xml

```diff
@@ -1,24 +1,25 @@
-<tool id="minfi_read450k" name="Minfi Read 450k" version="2.1.0">
+<tool id="minfi_read450k" name="Minfi Read 450k" version="2.1.1">
     <description>load .IDAT files</description>
     <requirements>
         <requirement type="package" version="1.24.0">bioconductor-minfi</requirement>
-    </requirements>
+    </requirements>    
     <command detect_errors="exit_code"><![CDATA[
-      #for $counter, $file in enumerate($files_red):
```

Comment:
@kpbioteam please let me know what you want to do here? I can hopefully help.

Reply:
Hi @bgruening I  want get/keep the file names of the files loaded to galaxy history. They change the names in temp dir (for 1,2,3 in our case) so they not really matching in my function anymore or if I try to overwrite the names they match randomly. Thanks